frontend/src/components/pages/CampOverview/CampersTable/CampersTableMenu.tsx

```diff
@@ -0,0 +1,31 @@
+import {
+  Menu,
+  MenuButton,
+  IconButton,
+  MenuList,
+  MenuItem,
+} from "@chakra-ui/react";
+import React from "react";
+import { FaEllipsisV } from "react-icons/fa";
+
+const CampersTableMenu = (): JSX.Element => {
```

Comment:
nit: thoughts on renaming to `CampersTableKebabMenu`, which makes it very explicit that it's the stacked dots menu?